lesson6-Reflection/Task2/CustomArgumentNullException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Task2
+{
+    public class CustomException : Exception
+    {
+        public CustomException(string message) : base(message)
```

Comment:
`MemberNotFoundException` sounds better for me. Also please don't forget to update the file name accordingly.

Reply:
done